commands/promote.js

```diff
@@ -50,6 +53,10 @@ function * run (context, heroku) {
     promotionMessage = `Promoting ${cli.color.addon(attachment.addon.name)} to ${cli.color.configVar('DATABASE_URL')} on ${cli.color.app(app)}`
   }
 
+  if (hasCurrent) {
+    cli.warn('Detaching DATABASE. If you are using the release phase feature (https://devcenter.heroku.com/articles/release-phase), pg:promote can cause a failed release. Please double checkthe releases and make sure the database is properly promoted.')
```

Comment:
@camillebaldock can we get +1 from product about this? tl;dr, we wanna warn the customer who is using release phase that `pg:promote` can do things that you don't expect. nobody reads warning messages these days but it's better to have than nothing IMO

@mehtaphysical I think we should drop "Detaching DATABASE" part as it can be confusing

Reply:
Can we avoid showing this to everyone who is not using release phase? It's totally irrelevant to them. If not, what's the release phase uptake? Is it worth warning release phase customers with a message that is not relevant to everyone else?

Reply:
@uhoh-itsmaciek yeah i think that's a good idea. I wasn't sure how to tell if someone was using release phase.

Reply:
maybe we could check if there are any pending or failed releases right after doing the promotion. If there are then we can display the warning?

Reply:
ok so we can either 1) display the warning to all release phase customers or 2) only display to release phase customers if there is a failed release during the detachment phase. I kind of like option 2 better.

Reply:
+1, but also check how trivial it is. I didn't actually expect this to be this wide scope, I'd say 1 is good enough if 2 gets complicated